styles/Typography.js

```diff
@@ -56,8 +56,18 @@ export default function Typography({ children, variant, component, ...rest }) {
 const StyledH1 = styled.h1``;
 const StyledH2 = styled.h2``;
 const StyledH3 = styled.h3``;
```

Comment:
```suggestion
```
This should not be in here, I think your linter should mention these empty parts.

Reply:
Adjusted accordingly to feedback